browser/src/canvas/sections/CommentListSection.ts

```diff
@@ -2375,7 +2375,7 @@ export class CommentSection extends app.definitions.canvasSectionObject {
 		if (this.sectionProperties.docLayer._docType === 'spreadsheet')
 			this.hideAllComments(); // Apply drawing orders.
 
-		if (!(<any>window).mode.isMobile() && (this.sectionProperties.docLayer._docType === 'presentation' || this.sectionProperties.docLayer._docType === 'drawing'))
```

Comment:
shouldn't be still non-mobile only? on mobile we have annotation mobile-wizard to manage comments